x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java

```diff
@@ -61,9 +60,14 @@
     private static final String NUMERICAL_FEATURE_FIELD = "feature";
     private static final String DISCRETE_NUMERICAL_FEATURE_FIELD = "discrete-feature";
     static final String DEPENDENT_VARIABLE_FIELD = "variable";
-    private static final List<Double> NUMERICAL_FEATURE_VALUES = List.of(1.0, 2.0, 3.0);
-    private static final List<Long> DISCRETE_NUMERICAL_FEATURE_VALUES = List.of(10L, 20L, 30L);
-    private static final List<Double> DEPENDENT_VARIABLE_VALUES = List.of(10.0, 20.0, 30.0);
+
+    // It's important that the values here do not work in a way where
+    // one of the feature is the average of the other features as it may
+    // result to empty feature importance and we want to assert it gets
+    // written out correctly.
+    private static final List<Double> NUMERICAL_FEATURE_VALUES = List.of(5.0, 2.0, 3.0);
```

Comment:
Just double-checking: is it ok that one feature is the sum of the other two (5 = 2 + 3) or could it also cause a problem?

Reply:
@valeriy42 What do you think?

Reply:
@przemekwitek this list defines the same feature (e.g. `x_1`) in three different rows. Rows are considered independently and therefore this is absolutely fine.  What you would like to avoid for this test is that one of the target values is the average of all three target values (previously we unfortunately had 20 == (10+20+30)/3 ).

What you probably also might have in mind is that different features in one row should be preferably linearly independen (e.g. x_1 = a* x_2 + b* x_3). This might lead to a problem but we use some techniques so that we don't have problem with linear dependency.

Reply:
Thanks for explanation @valeriy42 !